pkg/jobs/jobspb/jobs.proto

```diff
@@ -277,7 +277,9 @@ message ImportDetails {
     (gogoproto.customtype) = "github.com/cockroachdb/cockroach/pkg/util/uuid.UUID"
   ];
 
-  // next val: 25
+  int32 default_int_size = 25;
```

Comment:
lets add a comment:
"DefaultIntSize is the integer type that a "naked" int will be resolved to during the import. This is set based on the session variable `DefaultIntSize` when the import is planned.

Reply:
Done.